action/protocol/rewarding/protocol.go

```diff
@@ -387,7 +387,7 @@ func (p *Protocol) deleteStateV2(sm protocol.StateManager, key []byte) error {
 func (p *Protocol) settleSystemAction(
 	ctx context.Context,
 	sm protocol.StateManager,
-	act action.TxCommon,
+	act action.TxDynamicGas,
```

Comment:
#4396 extends this interface to `TxCommon`, with this change, we can reduce it back to `TxDynamicGas`